invenio_records_rest/views.py

```diff
@@ -117,6 +117,7 @@ def create_url_rules(endpoint, list_route=None, item_route=None,
                      delete_permission_factory_imp=None,
                      record_class=None,
                      record_serializers=None,
+                     record_serializers_query_aliases=None,
```

Comment:
Maybe just `(record_)serializers_aliases`, to keep it generic? (in case somebody wants to something fancy...)

Reply:
done!